Lib/_collections_abc.py

```diff
@@ -6,6 +6,32 @@
 Unit tests are in test_collections.
 """
 
+############ Maintenance notes #########################################
+#
+# ABCs are different from other standard library modules in that they
+# specify compliance tests.  In general, once an ABC has been published,
+# new methods (either abstract or concrete) cannot be added.
+#
+# Though classes that inherit from an ABC would automatically receive a
+# new mixin method, registered classes would become non-compliant and
+# violate the contract promised by ``isinstance(someobj, SomeABC)``.
+#
+# Though irritating, the correct procedure for adding new abstract or
+# mixin methods is to create a new class as a subclass of the previous
+# class.  For example, union(), intersection(), and difference() cannot
```

Comment:
I'd replace "class" with "ABC" in both occurrences in this sentence, assuming that's what you meant.

Reply:
Will do.